pkg/networkplugin-syncer/handlers/ovn/handler.go

```diff
@@ -17,14 +17,18 @@ var WaitingForLocalEndpoint = errors.New("Waiting for the local endpoint details
 
 type SyncHandler struct {
 	event.HandlerBase
-	syncMutex        sync.Mutex
-	k8sClientset     clientset.Interface
-	nbctl            *nbctl.NbCtl
-	nbdb             goovn.Client
-	sbdb             goovn.Client
-	localEndpoint    *submV1.Endpoint
-	remoteEndpoints  map[string]*submV1.Endpoint
-	lastOvnGwChassis string
+	syncMutex             sync.Mutex
+	k8sClientset          clientset.Interface
+	nbctl                 *nbctl.NbCtl
+	nbdb                  goovn.Client
+	sbdb                  goovn.Client
+	localEndpoint         *submV1.Endpoint
+	remoteEndpoints       map[string]*submV1.Endpoint
+	lastOvnGwChassis      string
+	submarinerUpstreamIP  string
+	submarinerUpstreamNET string
```

Comment:
I can’t suggest for some reason, but this should be `submarinerUpstreamNet`.

Reply:
Done.